packages/frontend/src/components/Card/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from "react";
+import { Works } from "types";
+import cn from "classnames";
+import Image from "next/image";
+
+import styles from "./styles.module.css";
+
+type CardProps = {
+  work: Works;
+}
+
+const Card:FC<CardProps> = ({work}) => {
+  const { icon, title, stack, summary } = work;
+  return(
+    <div className={cn(styles["card__container"],
+    "Trk__flex Trk__direction--column")}>
+      <div className={styles["card__logo"]}
+      style={{
+        backgroundImage:`url(${icon})`,
+        backgroundPosition: "center",
+        backgroundSize: "cover",
+        backgroundRepeat: "no-repeat",
+        width: "5rem",
```

Comment:
Reduce the width and height to something like 3rem. This is too. Also, let the margin-bottom be 1rem